File: Beecrowd/1858 A Resposta de Theon/main.cpp
#include <iostream>

using namespace std;

int obterPosMenor(int v[], int n);

int main(){
    int n, i;

    cin >> n;

    int v[n];

    for(i = 0; i < n; i++)
        cin >> v[i];

    cout << obterPosMenor(v, n) << endl;

    return 0;
}

int obterPosMenor(int v[], int n){
    int pos = 0, menor, i;
    menor = v[0];

    for(i = 0; i < n; i++){
        if (v[i] < menor){
            menor = v[i];
            pos = i;
        }
    }

    return pos + 1;
}

File: Beecrowd/1858 A Resposta de Theon/main.py
def obterPosMenor(v):
    v_min = (min(v)) # pega o menor valor da lista
    pos = v.index(v_min) + 1 # Pega a posição do menor valor da lista
    
    return pos

n = int(input())
inp = input().split() # entrada para lista

v = []

# considera só as entradas que esta de acordo com valor maximo que é n
for i in range(n):
    v.append(inp[i]) # add na lista

print(obterPosMenor(v))